src/lidar/instructions_bgflood.json

```diff
@@ -14,9 +14,9 @@
 			"number_of_cores": 3
 		},
 		"data_paths": {
-			"local_cache": "U:/Research/FloodRiskResearch/DigitalTwin/LiDAR/cache",
+			"local_cache": "U:/Research/FloodRiskResearch/DigitalTwin/stored_data/cache",
 			"subfolder": "results",
-			"catchment_boundary": "small_catchment.geojson",
```

Comment:
Yes but I want this path to be configured in one location, in this case the .env file. Do you know of a way to put this into here? This was the best solution I could come up with

Reply:
I'd just define it in one location then and write it into the dict you read in from instructions_bgflood.json at run time. I.E. do it dynamically.

Also if this instruction file is only creating a geofabric for use in BGFLOOD it might be better to rename 'instructions_geofabrics.json' or if it has info used by both processes might be better to give a wider name.

Reply:
Yup I am currently writing it in dynamically at run time, so I have removed it from the instructions file. I will rename it too